recipes/openmeeg/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = "2.4.0.dev" %}
+
+package:
+  name: openmeeg
+  version: {{ version }}
+
+source:
+  url: https://github.com/openmeeg/openmeeg/archive/2.4-rc1.tar.gz
```

Comment:
It's probably better to use a version / source_tag variable defined in the beginning of the file as suggested in https://github.com/conda-forge/staged-recipes/pull/5534#issuecomment-378941409
so that following updated can be done without changing the url, just the version tag.